docs/data.rst

```diff
@@ -52,7 +52,7 @@ Note that many things that you might use mapping for can also be done with
 Filtering
 ---------
 
-filter lets you reject some examples. ``s.filter(f).example()`` is some example
+Filter lets you reject some examples. ``s.filter(f).example()`` is some example
```

Comment:
I think the lower case is correct but this should be a literal. So "\`\`filter\`\`" rather than "Filter" or "filter"

Reply:
Corrected.

Reply:
I tried to make it consistent with other uses of 'map' and 'flatmap' in the surrounding text. Have made these all into symbols.